src/components/Navbar.jsx

```diff
@@ -0,0 +1,23 @@
+import { useNavigate } from 'react-router-dom';
+import TopMenuIcon from './MenuIcon';
+
+export default function Navbar() {
+    const navigate = useNavigate();
```

Comment:
Are you going to use `navigate` anywhere in this component?

Reply:
Deleted in code & Added TODO note